cmd/helm/search_repo_test.go

```diff
@@ -70,7 +70,7 @@ func TestSearchRepositoriesCmd(t *testing.T) {
 		golden: "output/search-output-json.txt",
 	}, {
 		name:   "search for 'maria', expect one match with semver begin with zero development version",
-		cmd:    "search repo maria --devel",
+		cmd:    "search repo maria --devel -l",
```

Comment:
why is this needed?

Reply:
Hrm. I seem to recall the output ordering to be unreliable, but I can't seem to reproduce it now. I'll drop this section.